.eslintrc.json

```diff
@@ -20,7 +20,7 @@
     "no-unused-vars": "warn",
     "no-var": "error",
     "prefer-const": "error",
-    "quotes": ["error", "single"],
+    "quotes": ["error", "single", {"avoidEscape": true}],
```

Comment:
Sneakily added this since I was playing w/ [**prettier**](http://npm.im/prettier) on this repo and it was doing some escaping of single quotes in the stories/index.js:

```
/Users/pdehaan/dev/github/mozilla/SnoozeTabs/stories/index.js
  27:12  error  Strings must use singlequote  quotes
  33:12  error  Strings must use singlequote  quotes
  63:12  error  Strings must use singlequote  quotes
  69:12  error  Strings must use singlequote  quotes

✖ 4 problems (4 errors, 0 warnings)
```